adapters/adhese/adhesetest/exemplary/banner-market.json

```diff
@@ -0,0 +1,114 @@
+{
+  "mockBidRequest": {
+    "id": "test-req",
+    "imp": [
+      {
+        "id": "test-req",
+        "banner": {
+          "format": [
+            {
+              "w": 300,
+              "h": 250
+            }
+          ]
+        },
+        "ext": {
+          "bidder": {
+            "account": "market",
+            "location": "_adhese_prebid_demo_",
+            "format": "leaderboard",
+            "targets": [
```

Comment:
It makes me wonder if we shouldn't consider making this structure more similar to the format that dale accepts...
```
      "ext": {
        "adhese": {
          "SL": "_dsp_-notexisting",
          "tl": [
            "all"
          ],
          "co": [
            "BE",
            "PL"
          ],
          "rg": [
            "BE:VWV"
          ],
          "dt": [
            "phone"
          ]
        }
      }
```
Although the account would not be compatible... so not 100% convinced by the idea.

But one thing we could do for sure is to make targets map more "kosher":
```{ "aa": ["123","456"] }```
instead of
```[{ "key": "aaa", "value": ["123","456"] }] ```

Reply:
I'll try this, but being able to type it comes with advantages:
`type AdheseKeywordsParams struct {
	Key    string   `json:"key,omitempty"`
	Values []string `json:"value,omitempty"`
}`

Reply:
We could adopt dale format and pass account in some other way @mefjush ? We just have to find a good alternative

Reply:
the account field is not such a big deal. Maybe the SL is more of a deal? Because then you'd need to change the current "location" and "format" to "SL"

Reply:
"location" and "format" have the benefit of being more readable to the user... So I don't know - maybe something to discuss, if we want to make the input format close to dale, or friendly to the user?

Reply:
Might be worth discussing. Seems better to decide now than to convince people to change their setup later :)